docs/interactions/structured-messages/template.md

```diff
@@ -113,3 +113,30 @@ Primary parameters are used by default, however, some parameters are unique or o
 | **`structured_content.items.title`** | Truncated to 25 characters. |
 | **`structured_content.items.payload`** | Automatically gets populated as a random hex if blank. |
 | **`structured_content.attachment_id`** | Supports private attachments. |
+
+## Example: WhatsApp (Reply Buttons)
+
+<img class="img-fluid" width="398" src="../../../img/structured-messages-template-whatsapp.png">
+
+### Properties Unique to this Channel
+
+Primary parameters are used by default, however, some parameters are unique or overwritten by parameters specific to this example.
+
+| API Property | Type | Specificity |
+|-|-|-|
+| **If subtitle is present** | | |
+| **`structured_content.title`** | String | Truncated to 60 characters. |
+| **`structured_content.subtitle`** | String | Truncated to 963 characters if an attachment is present. |
+|
+| **If subtitle is empty** | | |
```

Comment:
Looks like this doesn't render correctly, on markdown at least. Maybe I can check the end result somewhere?

![Screenshot from 2021-09-08 15-12-53](https://user-images.githubusercontent.com/6832100/132515965-c045a9e8-b271-4c7a-8c11-6a6ae63d2a53.png)

Reply:
fixed